poc/vdaf_prio3.py

```diff
@@ -483,6 +511,53 @@ class Prio3HistogramWithLength(Prio3Histogram):
         return Prio3HistogramWithLength
 
 
+# Experimental multiproof variant of Prio3SumVec
+class Prio3SumVecWithMultiproof(Prio3SumVec):
+    # Operational parameters.
+    test_vec_name = 'Prio3SumVecWithMultiproof'
+
+    @staticmethod
+    def is_recommended(valid_cls,
+                       num_proofs: Unsigned,
+                       field: FftField) -> bool:
+        if num_proofs >= 0x100:
+            # Currently we encod this as one byte while deriving joing rands
+            return False
+        # To be confirmed
+        if field == Field64:
+            return num_proofs >= 2
+        elif field == Field96:
```

Comment:
Do you plan to potentially use this field? Before you do, I think we should figure out if we can find a better prime modulus. We'd like to find something that's closer to a power of 2. (cc/ @divergentdave)

Reply:
No. So we will limit to `Field64` only then?

Reply:
Yes. Better to not use `Field96` unless we have to :)